### DISCUSSION TITLE
EbWizardry-related tweak for EntityLivingBase.isEntityUndead() check

### DISCUSSION THREAD

**Discussion by WinDanesz** [ACGaming/UniversalTweaks#148] [Tweak Suggestions]
Electroblob's Wizardry adds a curse potion effect, Curse of Undeath. The mod itself doesn't depend on mixins so I wouldn't put this in the mod, but it would be a fun addition to have the `EntityLivingBase.isEntityUndead()` method have a check whether this potion is active on the entity and return true if so.

**Comment by ACGaming** [comment]
Handled in Ancient Spellcraft.
